#Drive
string name
string type
float32 updateIntervalS
float32 cmdTimeoutS
float32 maxVelocity
float32 minPosition
float32 maxPosition

#SimDrive
float32 initialPosition
float32 positionPGain
float32 positionDGain
float32 velocityTimeToTargetS